DemoApp/StreamChat/Components/CustomAttachments/LocationAttachment/LocationAttachmentSnapshotView.swift

```diff
@@ -3,21 +3,45 @@
 //
 
 import MapKit
+import StreamChat
 import StreamChatUI
 import UIKit
 
-class LocationAttachmentSnapshotView: _View {
-    static var snapshotsCache: NSCache<NSString, UIImage> = .init()
+class LocationAttachmentSnapshotView: _View, ThemeProvider {
```

Comment:
Do you think it's a lot of effort if we also do this in the SwiftUI SDK?

Reply:
Most of the code is for the Demo App, so it might take some time. Although most of the stuff can be reused from UIKit I believe 🤔